Tests/SpeziTests/DependenciesTests/DependencyTests.swift

```diff
@@ -84,21 +84,6 @@ private final class TestModule7: Module {
     @Dependency var testModule1 = TestModule1()
 }
 
-// Swift 6 compiler doesn't allow circular references (even when there is a property wrapper in between)
-// Review this in future versions.
-#if compiler(<6)
-private final class TestModuleCircle1: Module {
```

Comment:
Do I see this correct that we don't need that check anymore as the compiler handles that for us?

Reply:
At the beginning the compiler error just occurred with Xcode 16, but then also persisted with Xcode 15. Even in the CI (maybe installing the latest Swift 6 compiler affected the stable one as well?). But yes, **currently** this produces a compiler error with the current toolchains.